### DISCUSSION TITLE
Error with unknown annotations

### DISCUSSION THREAD

**Discussion by mutasDev** [github/codeql#11113] [Q&A] [answered]
When I try to execute queries that use some of the semmle imports, I get the following error:

ERROR: unknown annotation additional (/Users/mutas/.codeql/packages/codeql/python-all/0.6.2/semmle/python/dataflow/new/internal/DataFlowImpl.qll:1665,5-15)
(for multiple different lines, not just 1665 as in the example above)

It seems there are dependencies missing for my dependencies?

Any Insights on how to fix this?

**Comment by atorralba** [comment, marked as answer]
`additional` is a recently introduced annotation, and you need an updated version of the CodeQL CLI to work with the latest `codeql/main`. What version of the CodeQL CLI are you running?

**Reply by mutasDev** [reply]
Hey ! 
Im using codeql cli version  2.10.1

**Reply by atorralba** [reply]
Could you try updating to the latest one (currently 2.11.2)?

**Reply by mutasDev** [reply]
That seems to work, thanks a lot !
